audio-search/requirements.txt

```diff
@@ -1,9 +1,8 @@
-jina[hub,http]==0.5.5
+jina[hub,http]==0.9.20
 numpy
```

Comment:
Let's try to have specific versions?

Reply:
numpy is not needed here, since it is one of the main dependencies of jina

Reply:
Add specific versions and update requirement